db/migrate/20230504172751_create_bookings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBookings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bookings do |t|
+      t.references :session, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
+      t.boolean :canceled
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Can you edit this migration file to include the changes you made in the second one ?
Once done you can delete the second migration file, drop your db, create a new one migrate again and it should be good